client/packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/ItemCharts/ConsumptionHistory.tsx

```diff
@@ -35,12 +35,16 @@ export const ConsumptionHistory: React.FC<ConsumptionHistoryProps> = ({
   const tooltipFormatter = (
     value: number,
     name: string,
-    props: { payload?: { date: string; isHistoric: boolean } }
+    props: {
+      payload?: { date: string; isHistoric: boolean; isCurrent: boolean };
+    }
   ): [number, string] => {
     switch (name) {
       case 'consumption':
         const label = props.payload?.isHistoric
           ? t('label.consumption')
+          : props.payload?.isCurrent
```

Comment:
yes, I have complained about nested ternary operators in other PRs. Yes, I feel bad doing it. Yes, I've done it anyway.